app/exerlog/lib/UI/calendar/calendar_page.dart

```diff
@@ -16,40 +19,81 @@ class _CalendarPageState extends State<CalendarPage> {
   Widget build(BuildContext context) {
     return Scaffold(
       backgroundColor: backgroundColor,
-      body: Container(
-        padding: EdgeInsets.only(top:30, left: 20, right: 20),
-        color: backgroundColor,
+      appBar: AppBar(
+        title: Text("Exerlog"),
+        backgroundColor: backgroundColor.withOpacity(0.75),
+        actions: [
+          _Logout(),
+        ],
+      ),
+      body: Padding(
+        padding: EdgeInsets.all(20),
         child: Column(
           mainAxisAlignment: MainAxisAlignment.spaceBetween,
           children: [
             CalendarWidget(),
             Container(
-              width: MediaQuery.of(context).size.width *0.8,
-              margin: EdgeInsets.only(bottom: 30),
+              width: MediaQuery.of(context).size.width * 0.8,
               child: RaisedGradientButton(
-                gradient: LinearGradient(
-                  colors: <Color>[Color(0xFF34D1C2), Color(0xFF31A6DC)],
-                ),
-                radius: 30,
                 child: Text(
                   "START NEW WORKOUT",
                   style: buttonTextSmall,
                 ),
                 onPressed: () {
-                  Navigator.of(context).pop();
-                  Navigator.of(context).push(
-                      MaterialPageRoute(
-                        builder: (context) => WorkoutPage(null
-                        ),
-                      ),
-                    );
+                  Navigator.of(context).pushReplacement(
+                    MaterialPageRoute(
+                      builder: (context) => WorkoutPage(null),
+                    ),
+                  );
                 },
               ),
             )
           ],
-          ),
+        ),
       ),
     );
   }
+}
+
+class _Logout extends StatefulWidget {

```

Comment:
Please refactor this widget into a seperate file like 'logout_button.dart'.
Also change the widget name into the same name as the file like 'LogoutButton'.